spikeinterface/core/curationsorting.py

```diff
@@ -0,0 +1,119 @@
+from .mergesorting import MergeSorting
+from .splitsortingunit import SplitSortingUnit
+import networkx as nx
+from collections import namedtuple
+node_t = namedtuple('node_t', 'id stage')
+
+class CurationSorting():
+    """
+    Class that handles curation of a Sorting object.
+
+    """
+
+    def __init__(self, parent_sorting):
+        #to allow undo and redo a list of sortingextractors is keep
+        self._sorting_stages = [parent_sorting]
+        self._sorting_stages_i = 0
+        parent_units = parent_sorting.get_unit_ids()
+        #to easily allow undo and redo a list of graphs with the history of the curation is keep
+        self._graphs = [nx.DiGraph()]
+        self._graphs[0].add_nodes_from([node_t(u,0) for u in parent_units])
+
+        self._kwargs = dict(parent_sorting=parent_sorting.to_dict())
+
+    def _get_unused_id(self):
+        #check units in the graph to the next unused unit id
+        nodes = [n.id for n in self._graphs[self._sorting_stages_i].nodes()]
+        return max(nodes)+1
+
+    def split(self,unit2split, indices_list):
+        current_sorting = self._sorting_stages[self._sorting_stages_i]
+        new_id = self._get_unused_id()
+        new_units_ids = [new_id, new_id+1]
+        new_sorting = SplitSortingUnit(current_sorting, unit2split=unit2split, indices_list=indices_list, new_units_ids = new_units_ids)
+        units = current_sorting.get_unit_ids()
+        i = self._sorting_stages_i
+        edges = [(node_t(u,i),node_t(u,i+1)) for u in units if u !=unit2split]
+        edges = edges + [(node_t(unit2split,i), node_t(u,i+1)) for u in new_units_ids]
+        self._add_new_stage(new_sorting, edges)
+
+    def merge(self,units2merge, rm_dup_delta_time=0.4, cache=False):
```

Comment:
```suggestion
    def merge(self,units2merge, delta_time=0.4, cache=False):
```
and explain in docstring